git-updater/templates/cleanup.sh.j2

```diff
@@ -0,0 +1,6 @@
+#!/bin/bash
+GIT_PATH=$1
+LOG_DIR=$2
```

Comment:
we don't seem to use any of the logging which i think we should.

Something that indicates when it started, when it stopped maybe even what the diff was?

Reply:
done in https://github.com/wmde/ansible-roles/pull/2/commits/f6589a795b7e46ea7cb8450ba3cc148de0dfeabc